tslib must be a dependency and not a devDependency

When using `importHelpers: true`, tslib must be a runtime dependency, currently it is a development dependency so importing `@dojo/core` throws an error `Error: Cannot find module 'tslib'`

Pull introducing error: #367

Why is this labeled `question`. This is definitely a bug!
Let me try to make it easier to understand. Please create a new directory and run the following:
```
$ npm i @dojo/core @dojo/has @dojo/shim --no-package-lock --global-style

$ node -e "require('@dojo/core/async/Task')"

Error: Cannot find module 'tslib'
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.Module._load (module.js:476:23)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
    at \node_modules\@dojo\core\async\Task.js:12:19
    at Object.defineProperty.value (\node_modules\@dojo\core\async\Task.js:3:17)
    at Object.<anonymous> (\node_modules\@dojo\core\async\Task.js:9:3)
    at Module._compile (module.js:624:30)
    at Object.Module._extensions..js (module.js:635:10)
    at Module.load (module.js:545:32)
```

The source code of `@dojo/core` contain `require('tslib')` statements so how could `tslib` not be a dependency of `@dojo/core`?

--- a/package.json
+++ b/package.json
@@ -22,6 +22,9 @@
     "prettier": "prettier --write 'src/**/*.ts' 'tests/**/*.ts'",
     "test": "grunt test"
   },
+  "dependencies": {
+    "tslib": "~1.8.1"
+  },
   "peerDependencies": {
     "@dojo/has": "~0.1.1",
     "@dojo/shim": "~0.2.3"
@@ -55,7 +58,6 @@
     "prettier": "1.9.2",
     "remap-istanbul": ">=0.6.3",
     "sinon": "~1.17.6",
-    "tslib": "~1.8.0",
     "tslint": "5.8.0",
     "tslint-plugin-prettier": "1.3.0",
     "typescript": "~2.6.1"

